components/ui/autocomplete/src/main/java/mozilla/components/ui/autocomplete/InlineAutocompleteEditText.kt

```diff
@@ -263,6 +265,29 @@ open class InlineAutocompleteEditText @JvmOverloads constructor(
         }
     }
 
+    private fun pasteAsPlainText(clipData: ClipData) {
+        var paste = ""
+        for (i in 0 until clipData.itemCount) {
+            val text = clipData.getItemAt(i).coerceToText(context)
+            paste += if (text is Spanned) text.toString() else text
+        }
+        val data = text.substring(0, selectionStart) + paste + text.substring(selectionEnd, text.length)
+        setText(data)
+    }
```

Comment:
We could consider moving this into `support-ktx` as an extension method, e.g. `ClipData.pasteAsPlainText()` (+ unit tests).